maven-dependency-plugin reference to 1.4-SNAPSHOT hapi-fhir-cli-jpaserver breaks build

"mvn clean install" fails, with the following message:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.10:copy (copy) on project hapi-fhir-cli-app: Unable to find artifact. Failure to find ca.uhn.hapi.fhir:hapi-fhir-cli-jpaserver:war:1.4-SNAPSHOT in (local repo url) was cached in the local repository, resolution will not be reattempted until the update interval of nexus has elapsed or updates are forced

--- a/hapi-fhir-cli/hapi-fhir-cli-app/pom.xml
+++ b/hapi-fhir-cli/hapi-fhir-cli-app/pom.xml
@@ -208,7 +208,7 @@
 								<artifactItem>
 									<groupId>ca.uhn.hapi.fhir</groupId>
 									<artifactId>hapi-fhir-cli-jpaserver</artifactId>
-									<version>1.4-SNAPSHOT</version>
+									<version>1.5-SNAPSHOT</version>
 									<type>war</type>
 									<overWrite>true</overWrite>
 									<outputDirectory>target/classes</outputDirectory>

